lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -16,33 +17,76 @@
 HTTPTIMEOUT = (3.05, 1)
 
 
+# Duplicated in delivery_status_processor.
+def validate_twilio_event(event):
+    logger.info("validating twilio vetext forwarder event")
+    try:
+        ssm_client = boto3.client("ssm", "us-gov-west-1")
+        auth_ssm_key = os.getenv("TWILIO_AUTH_TOKEN_SSM_NAME", "")
+        if not auth_ssm_key:
+            logger.error("TWILIO_AUTH_TOKEN_SSM_NAME not set")
+            return False
+
+        response = ssm_client.get_parameter(
+            Name=auth_ssm_key,
+            WithDecryption=True
+        )
+        auth_token = response.get("Parameter").get("Value")
+        signature = event["headers"].get("x-twilio-signature", "")
+    except Exception as e:
+        logger.error("SMS retrival error %" % e)
+        logger.error(e)
+        return False
+
+    try:
+        if not auth_token or not signature:
+            logger.error("TWILIO_AUTH_TOKEN or signature not set")
+            return False
+
+        validator = RequestValidator(auth_token)
+        uri = "https://%s/vanotify/twoway/vettext" % event["headers"]["host"]
+        logger.info(f"URI: {uri}")
+
+        return validator.validate(
+            uri=uri,
+            params=event["body"],
+            signature=signature
+        )
+    except Exception as e:
+        logger.error("Twilio library error: %" % e)
+        logger.error(e)
+        return False
+
+
 def vetext_incoming_forwarder_lambda_handler(event: dict, context: any):
     """this method takes in an event passed in by either an alb or sqs.
         @param: event   -  contains data pertaining to an incoming sms from Twilio
         @param: context -  contains information regarding information
             regarding what triggered the lambda (context.invoked_function_arn).
     """
-
     try:
-        logger.debug(event)
+        logger.info(event)
```

Comment:
we probably want to leave events and event bodies as debug logs

Reply:
Now that you're done testing, can this be set to debug?

Reply:
```suggestion
        logger.debug("Entrypoint event: %s", event)
```
Agreed with @cris-oddball . Additionally, logging just "event" is not sufficient. It can make reading the logs quite difficult if in other places they log other dictionaries without qualifiers.